content/02.intro.md

```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the characterization of genomic and biological data [@doi:10.1098/rsif.2017.0387].
+Despite its growing popularity, DL itself remains an active area of research, and its everchaning complexity and lack of current beginner resources pose large barriers of entry to newcomers who wish to utilize state-of-the-art DL in their research.
```

Comment:
Afaik there's tons and tons of beginner resources for DL (more than for most other field/methods). Maybe we should rephrase this a bit to specifically mean DL applications to biological problems, because there, beginner resources are indeed a bit scarce and practically non-existent aside from short method sections in research papers